src/EnergyPlus/EconomicTariff.cc

```diff
@@ -3864,51 +3864,63 @@ void LEEDtariffReporting(EnergyPlusData &state)
     Real64 elecTotalEne;
     Real64 gasTotalEne;
     Real64 distCoolTotalEne;
-    Real64 distHeatTotalEne;
+    Real64 distHeatWaterTotalEne;
+    Real64 distHeatSteamTotalEne;
     Real64 otherTotalEne;
     Real64 elecTotalCost;
     Real64 gasTotalCost;
     Real64 otherTotalCost;
     Real64 distCoolTotalCost;
-    Real64 distHeatTotalCost;
+    Real64 distHeatWaterTotalCost;
+    Real64 distHeatSteamTotalCost;
     Real64 allTotalCost;
+    std::string distCoolTariffNames;
+    std::string distHeatWaterTariffNames;
+    std::string distHeatSteamTariffNames;
     EconConv elecUnits;
     EconConv gasUnits;
     EconConv distCoolUnits;
-    EconConv distHeatUnits;
+    EconConv distHeatWaterUnits;
+    EconConv distHeatSteamUnits;
     EconConv othrUnits;
     DemandWindow gasDemWindowUnits;
     DemandWindow distCoolDemWindowUnits;
-    DemandWindow distHeatDemWindowUnits;
+    DemandWindow distHeatWaterDemWindowUnits;
+    DemandWindow distHeatSteamDemWindowUnits;
     DemandWindow othrDemWindowUnits;
 
     auto const &tariff = state.dataEconTariff->tariff;
 
     if (state.dataEconTariff->numTariff > 0) {
         int distCoolFacilMeter = GetMeterIndex(state, "DISTRICTCOOLING:FACILITY");
-        int distHeatFacilMeter = GetMeterIndex(state, "DISTRICTHEATING:FACILITY");
+        int distHeatWaterFacilMeter = GetMeterIndex(state, "DISTRICTHEATINGWATER:FACILITY");
+        int distHeatSteamFacilMeter = GetMeterIndex(state, "DISTRICTHEATINGSTEAM:FACILITY");
```

Comment:
For transition, do we assume that all existing districtheating is transitioned to DistrictHeatingWater?

Reply:
Yes for now, all existing `Districtheating` is transitioned to `DistrictHeatingWater` since existing `Districtheating` is about water loop, but if you have any concerns/suggestions, please let me know.